posts/2020-08-16-ocp-log-forwarding.adoc

```diff
@@ -0,0 +1,421 @@
+---
+layout: post
+title: "Forwarding Open Liberty logs in OpenShift Container Platform to Splunk using Log Forwarding API"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/leemelim
+author_github: https://github.com/leemelim
+seo-title: Forwarding Open Liberty logs in OpenShift Container Platform to Splunk using Log Forwarding API - OpenLiberty.io
+seo-description: Using Log Forwarding API, you can send logs from Open Liberty deployed in OpenShift Container Platform to remote destinations. You can deploy Splunk on your external machine or inside your OpenShift Container Platform. The logs can be forwarded to Splunk using the Fluentd forward protocol.
+blog_description: Using Log Forwarding API, you can send logs from Open Liberty deployed in OpenShift Container Platform to remote destinations. You can deploy Splunk on your external machine or inside your OpenShift Container Platform. The logs can be forwarded to Splunk using the Fluentd forward protocol.
+---
+= Forwarding Open Liberty logs in OpenShift Container Platform to Splunk using Log Forwarding API
+Halim Lee <https://github.com/leemelim>
+
+OpenShift Container Platform provides a log aggregation solution with cluster logging using Elasticsearch, Fluentd, and Kibana (EFK stack). In addition to EFK stack, integration with other logging applications has been made available. Starting OpenShift 4.3, you can use Log Forwarding API, which is currently in Technology preview, to send logs from Open Liberty in OpenShift cluster to other available log analysis solutions.
```

Comment:
In addition to EFK stack, integration with other logging applications has been made available
->
In addition to EFK stack, you can integrate with other log analysis solutions